patroni/dcs/zookeeper.py

```diff
@@ -126,6 +126,9 @@ def __init__(self, config: Dict[str, Any]) -> None:
         self._client._connection._connect = self._kazoo_connect
 
         self._client.start()
+        ensemble_auth = config.get('ensemble_auth', '')
+        if ensemble_auth:
+            self._client.add_auth("ensemble", ensemble_auth)
```

Comment:
The thing is that the python-kazoo module doesn't have a single mention of "ensemble" in this context.
Therefore lets be general enough: I would prefer passing `auth_data` parameter to the `KazooClient` constructor, similar to `set_acls`, instead of hardcoding "ensemble_auth".
In the config it may look like:
```yaml
zookeeper:
  auth_data:
    ensemble: blabla
```
or env variable:
```
PATRONI_ZOOKEEPER_AUTH_DATA='{auth_data: {ensemble: blabla}}'
```

Reply:
👍  done